src/elf.c

```diff
@@ -464,11 +464,10 @@ int elf_validate_modules(struct image *image)
 	return 0;
 }
 
-int elf_find_section(struct image *image, struct module *module,
-		     const char *name)
+int elf_find_section(const struct module *module, const char *name)
 {
 	Elf32_Ehdr *hdr = &module->hdr;
-	Elf32_Shdr *section, *s;
+	const Elf32_Shdr *section, *s;
```

Comment:
This addition of "const" would normally warrant a separate commit just for itself. I won't impose this though.